File: CFOnlineShop/CFOnlineShop/ViewController/CFClassificationController/CFClassificationCell.m
//
//  CFClassificationCell.m
//  CFOnlineShop
//
//  Created by chenfeng on 2018/7/24.
//  Copyright © 2018年 chenfeng. All rights reserved.
//

#import "CFClassificationCell.h"

@implementation CFClassificationCell

- (instancetype)initWithFrame:(CGRect)frame
{
    self = [super initWithFrame:frame];
    if (self) {
        
        self.backgroundColor = kWhiteColor;
        
        _imageView = [[UIImageView alloc] initWithFrame:CGRectMake(0, 0,self.mj_w, self.mj_w)];
        _imageView.contentMode = UIViewContentModeScaleAspectFit;
        [self.contentView addSubview:_imageView];
        
        _titleStr = [[UILabel alloc] initWithFrame:CGRectMake(15, self.mj_w + 15, self.mj_w - 30, 20)];
        _titleStr.font = SYSTEMFONT(14);
        _titleStr.textColor = KDarkTextColor;
        [self.contentView addSubview:_titleStr];
        
    }
    
    return self;
}

@end

File: SwiftOnlineShopDemo/SwiftOnlineShopDemo/ViewController/CFClassificationController/view/CFClassificationCell.swift
//
//  CFClassificationCell.swift
//  SwiftOnlineShopDemo
//
//  Created by chenfeng on 2019/4/29.
//  Copyright © 2019年 chenfeng. All rights reserved.
//

import UIKit

class CFClassificationCell: UICollectionViewCell {
    
    var imageView: UIImageView!
    var titleStr: UILabel!
    
    override init(frame: CGRect) {
        
        super.init(frame: frame)
        
        self.backgroundColor = UIColor.white
        
        imageView = UIImageView.init(frame: CGRect(x: 0, y: 0, width: self.mj_w, height: self.mj_w))
        imageView.contentMode = .scaleAspectFit
        self.contentView.addSubview(imageView)
        
        titleStr = UILabel.init(frame: CGRect(x: 15, y: self.mj_w + 15, width: self.mj_w - 30, height: 20))
        titleStr.font = UIFont.systemFont(ofSize: 14)
        titleStr.textColor = UIColor.black
        self.contentView.addSubview(titleStr)
        
    }
    
    required init?(coder aDecoder: NSCoder) {
        fatalError("init(coder:) has not been implemented")
    }
    
}
